core/Piranha.Manager/assets/src/js/piranha.antiforgery.js

```diff
@@ -0,0 +1,18 @@
+﻿/*global
+    piranha
+*/
+
+piranha.antiforgery = new function () {
+    "use strict";
+
+    var self = this;
+
+    this.init = function () {
+        if (document.getElementsByName('__RequestVerificationToken').length > 0)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PiranhaCMS/piranha.core/pullRequest?prid=7901588)
```suggestion
        if (document.getElementsByName"__RequestVerificationToken"').length > 0)
```